src/web/mjs/connectors/PiedPiper.mjs

```diff
@@ -4,9 +4,9 @@ export default class PiedPiper extends WordPressMadara {
 
     constructor() {
         super();
-        super.id = 'piedpiperfb';
+        super.id = 'piedpiperfansub';
```

Comment:
`ID` should not be changed, bookmarks of the users will become invalid and need to be re-bookmarked.
Also if this is really necessary, the icon must be renamed to match the new `ID`.

Reply:
OK, I would leave the Id unchanged then